translations/monero-core_de.ts

```diff
@@ -1079,235 +1079,241 @@ The following information will be deleted
 
 The old wallet cache file will be renamed and can be restored later.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Bist Du sicher, dass Du den Wallet-Cache neu erstellen möchtest?
+Die folgenden Informationen werden gelöscht
+- Empfängeradressen
+- Tx-Schlüssel
+- Tx-Beschreibungen
```

Comment:
I figured out what a Tx is after the second mention on a post on reddit; I guess with a person that's comfortable in english, the "x" for "act" to "action" comes natural because of XMas (Crossmas -> Chrismas), but that would be a proposal for a different PR and should be discussed before proposal.

I'm personally happy with Tx in the english localization.